exams/management/commands/simulate_sftp_responses.py

```diff
@@ -0,0 +1,158 @@
+"""Command to simulate Pearson responses"""
+import csv
+import io
+import random
+import time
+import zipfile
+from datetime import datetime
+
+import pytz
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from exams.pearson.constants import PEARSON_DATETIME_FORMAT
+from exams.pearson import sftp
+
+
+class Command(BaseCommand):
+    """Simulates Pearson responses"""
+    help = 'Simulates the SFTP backend by generating random responses'
+
+    def add_arguments(self, parser):  # pylint: disable=no-self-use
+        """Configure command args"""
+        parser.add_argument(
+            '--ratio-success',
+            dest='ratio',
+            type=float,
+            default=0.5,
+        )
+        parser.add_argument(
+            '--interval',
+            dest='interval',
+            type=int,
+            default=5,
+        )
+        parser.add_argument(
+            '--poll',
+            action='store_true',
+            dest='poll',
+            default=False,
+        )
+        parser.add_argument(
+            '--keep-files',
+            action='store_true',
+            dest='keep',
+            default=False,
+        )
+
+    def handle(self, *args, **options):
+        """Handle the command"""
+        ratio = options['ratio']
+        poll = options['poll']
+        interval = options['interval']
+        keep = options['keep']
+
+        if poll:
+            while True:
```

Comment:
There isn't an easy way to terminate this in a test. I think there are some packages that implement timeout-based short-circuiting but I don't think all that is worth pulling in just to test this small section of code.